Class1.cs

```diff
@@ -0,0 +1,22 @@
+﻿// This is the test class which will work with the Projector Abstract Class 
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace RS232_API
+{
+    class Class1
+    {
+        static void Main(string[] args)

```

Comment:
This class isn't actually testing anything. Read about unit testing in C#. But you know what given how the abstract classes work in C#, you could just remove the whole test file.

Reply:
I read about unit testing. I created a unit test. The test class is currently just a skeleton of the Projector Class since the actual class and methods are empty. Is this okay?

Reply:
I also had to make the Projector class public since a unit test only works with public classes.

Reply:
This was good practise for writing a unit test but for an abstract class it's unnecessary so you can just remove the whole test class and only test non abstract classes

Reply:
I removed the file. I don't have a non-abstract class. I don't understand what class needs a test class.